Makefile

```diff
@@ -10,6 +10,7 @@ SUBDIR += verifier
 SUBDIR += frontend
 SUBDIR += cmd
 SUBDIR += kvstore
+SUBDIR += provisioning
```

Comment:
There is a typo on line 15 (frontend), just tracking it here for reference:

Reply:
what is the typo?

Reply:
It is on line 15 I suppose, not in your code

Reply:
frontentd